common/djangoapps/track/middleware.py

```diff
@@ -70,7 +83,12 @@ def process_request(self, request):
 
             views.server_track(request, request.META['PATH_INFO'], event)
         except:
-            pass
+            ## Why do we have the overly broad except?
+            ##
+            ## Adding instrumentation. If we're dropping events on the floor,
+            ## we should know about it.
+            event = {'event-type': 'exception', 'exception': sys.exc_info()[0]}
```

Comment:
I want this in the standard tracking logs so users downstream (e.g. me, researchers, researchers in two decades looking over archives) can know if there is missing data. The standard exception logs essentially go to /dev/null most of the time. They're not archived, and they're not even really examined except to debug specific issues.